src/libmongoc/src/mongoc/mongoc-openssl.c

```diff
@@ -76,7 +76,9 @@ _mongoc_openssl_init (void)
 
    SSL_library_init ();
    SSL_load_error_strings ();
+#if OPENSSL_VERSION_NUMBER < 0x30000000L
```

Comment:
```suggestion
#if OPENSSL_VERSION_NUMBER < 0x30000000L
   // See "deprecated function mappings" in:
   // https://www.openssl.org/docs/man3.0/man7/migration_guide.html
```
Suggest including a reference to corresponding docs.

Reply:
Are `//` comments kosher in libmongoc? AFAIK the project uses `/* ... */` exclusively.

Reply:
Use of `//` had been restricted due to C89/C90 compatibility due to being a C99 feature. Given progress on CDRIVER-4136 we have begun freely using this comment style throughout the repo.

I personally prefer documentation to be as local as possible, but I would not be opposed to pushing it into the commit message instead (as rationale is still captured). I will leave that up to you.

Reply:
Added a one-line comment with a link to the docs.